### PR TITLE
Update collection-index.yml - Add GitHub Actions runner devcontainer template 

### PR DESCRIPTION
**Pull request by Pwd9000-ML**
Add GH-runner devcontainer template

### CODE REVIEW

**Comment by Pwd9000-ML** [comment]
I've started work on a set of devcontainer templates that I have created and used in my personal codespaces to run a GitHub actions runner within the compute of a Codespace to use as part of a Codespace, I have a similar devcontainer templates for Azure DevOps agents I will add at a later date.
I find these have proven very useful and thus wanting to share and contribute my work as a template to be used by others.
Would love to hear some thoughts on the devcontainer :)

File: _data/collection-index.yml [hunk 1]
```diff
@@ -243,3 +243,8 @@
   contact: https://github.com/ChristopherMacGown/devcontainer-features/issues
   repository: https://github.com/ChristopherMacGown/devcontainer-features
   ociReference: ghcr.io/christophermacgown/devcontainer-features
+  - name: Pwd9000-ML devcontainer templates
+  maintainer: Marcel Lupo @Pwd9000-ML
+  contact: https://github.com/Pwd9000-ML/devcontainer-templates/issues
+  repository: https://github.com/Pwd9000-ML/devcontainer-templates
+  ociReference: ghcr.io/Pwd9000-ML/devcontainer-templates
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/pwd9000-ml/devcontainer-templates
```

The ORAS spec requires the `ociReference` to be in lower case. Can we update here? 

The supporting tools does help convert the cases, however, its better to sync as much as possible.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
The template looks awesome & helpful. Thank you! 

Left one comment.

**Comment by Pwd9000-ML** [comment]
@samruddhikhandale I've updated the `ociReference:` to all lowercase. Thank you for the recommendation.

**Review by samruddhikhandale — APPROVED** [review]
Awesome, thanks!

**Pull request merged**